deploy/helm/sumologic/values.yaml

```diff
@@ -211,6 +211,7 @@ fluentd:
     totalLimitSize: "128m"
     retryMaxInterval: "10m"
     retryForever: true
+    compress: text
```

Comment:
Should we add a comment that GZIP is not supported so it is clear why we set it this way?

Reply:
@frankreno I would say not recommended and point to the issue. Is it ok?